src/main/java/org/btcprivate/wallets/fullnode/util/Util.java

```diff
@@ -320,7 +320,8 @@ public static String convertGroupPhraseToZPrivateKey(String phrase)
     public static boolean isZAddress(String address)
     {
         return (address != null) &&
-                address.startsWith("zk") &&
+                (address.startsWith("zk")) &&
+//                (address.startsWith("zk") || address.startsWith("zz") || address.startsWith("zt")) &&
```

Comment:
did this sneak in? or do we need this for testnets to comment it out?

Reply:
The commented out line was kept in simply for convenience- for future testing w/ regtest/testnet addresses. I'm fine with leaving it in or taking it out as well.